packages/helm-charts/oracle/values.yaml

```diff
@@ -0,0 +1,4 @@
+nodeSelector: {}
+
+# Expects cron schedule syntax
+oracleSchedule: "*/1 * * * *"
```

Comment:
might be helpful to add a comment what this schedule means in readable terms for people like myself that never remember the cron schedule structure

Reply:
Excellent idea. I had to look it up myself.
Do you think this makes sense here, or should it be an envVar like all the other ones?